lib/puppet/util/docs.rb

```diff
@@ -96,7 +96,7 @@ def markdown_definitionlist(term, definition)
   # formatting of the code they're embedded in. Since indents are syntactic
   # elements in Markdown, we need to make sure we remove any indent that was
   # added solely to preserve surrounding code formatting, but LEAVE any indent
-  # that delineates a Markdown element (code blocks, multi-line bulleted list
```

Comment:
this was not wrong

Reply:
@l0b0 can you change this one back to "bulleted" please